content/pl/user/profile.md

```diff
@@ -0,0 +1,77 @@
+---
+title: Dostosuj swój profil
+description: Rozpocznij korzystanie z nowego konta.
+menu:
+  docs:
+    weight: 20
+    parent: user
+---
+
+## Wygląd Twojego profilu {#appearance}
+
+{{< figure src="/assets/image%20%2829%29.png" caption="Karta profilu składająca się z nazwy wyświetlanej, awataru i nagłówka" >}}
+
+Możesz zmienić to, jak Twój profil wygląda u innych, kierując się do: Preferencje &gt; Profil &gt; Wygląd.
+
+### Nazwa wyświetlana {#name}
+
+Twoja nazwa wyświetlana jest widoczna dla innych przy Twoim adresie. Domyślnie możesz ustawić nazwę zawierającą do 30 znaków.
+
+### Biogram {#bio}
+
+Twój biogram to krótki opis Twojej osoby wyświetlana jako notatka na Twoim profilu. Domyślnie możesz ustawić biogram składający się maksymalnie z 500 znaków.
+
+### Awatar {#avatar}
+
+Twój awatar to ikona wyświetlana po lewej od Twoich wpisów i część Twojej wizualnej tożsamości. Możesz wysłać obraz PNG, GIF lub JPG w wielkości do 2MB jako awatar. Zostanie on zmniejszony do 400x400.
+
+### Nagłówek {#header}
+
+Twój nagłówek to baner wyświetlany na górze Twojego profilu, jak i na kartach profilu wyświetlanych w listach obserwacji czy katalogu profili. Możesz wysłać obraz PNG, GIF lub JPG w wielkości do 2MB jako nagłówek. Zostanie on zmniejszony do 1500x500.
+
+## Flagi profilu {#flags}
+
+Możesz oflagować swoje konto, aby poinformować innych, w jaki sposób korzystasz z Mastodona.
+
+![](/assets/image%20%281%29.png)
+
+### Konto zablokowane {#locked}
+
+Jeśli zablokujesz swoje konto, staną się dwie rzeczy:
+
+* Nowi obserwujący nie będą automatycznie przyjmowani, będziesz musiał(a) ręcznie zatwierdzać ich.
+* Ikona kłódki będzie widoczna dla innych, aby wiedzieli, że obserwacja nie zostanie automatycznie przyjęta.
+
+### Konto bota {#bot}
+
+Włączenie tej flagi doda ikonkę bota do Twojego profilu. Dzięki temu inni dowiedzą się, że Twój profil może dokonywać zautomatyzowanych działań i może nie być nadzorowany przez człowieka. Inne oprogramowanie może traktować profile botów w inny sposób, ale obecnie Mastodon jedynie wyróżnia boty wizualnie.
+
+### Katalog profilów {#discoverable}
+
+Udział w katalogu profilów sprawi, że Twój profil będzie mógł zostać odnaleziony przez tę funkcję, pozwalającą na przeglądanie profilów.
+
+## Metadane profilu {#fields}
+
+Metadane profilu to możliwość dodania do profilu dodatkowych, przejrzystych informacji. Masz cztery wiersze, w których możesz wpisać podpis i zawartość pola. Na przykład:
+
+| Podpis | Zawartość |
```

Comment:
```suggestion
| Etykieta | Zawartość |
```

Reply:
Actually, it uses *nazwa* | *zawartość* in the UI, so I'll use these terms